docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Active.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "org.fwupd.hsi.IntelCet.Active",
-  "name": "Intel CET: Active",
+  "name": "CET Utilized by OS",
```

Comment:
I don't like the `Utilized` word -- maybe `CET In Use`? or `CET OS Support`?

Reply:
Makes sense to me.

Reply:
or "CET Hardware Support" perhaps?

Reply:
the reason I wanted to use platform is that it's not it's not impossible that CPU microcode or BIOS microcode influences the ability for it to work.  But if you think `hardware support` is better I'll swap to that.

Reply:
Platform is probably more correct, agreed.